### ISSUE TITLE
[Bug]: unexpected state PENDING during aws_mwaa_environment creation

### ISSUE THREAD

**Issue by maxmuffin** [hashicorp/terraform-provider-aws#37223]
### Terraform Core Version

1.5.0

### AWS Provider Version

5.39.0

### Affected Resource(s)

aws_mwaa_environment

### Expected Behavior

resource creation without errors 

### Actual Behavior

resource creation with error: 
_waiting for MWAA Environment (airflow-environment) create: unexpected state 'PENDING', wanted target 'AVAILABLE', last error: %!s(<nil>)_

### Relevant Error/Panic Output Snippet

```shell
waiting for MWAA Environment (airflow-environment) create: unexpected state 'PENDING', wanted target 'AVAILABLE', last error: %!s(<nil>)
```

### Terraform Configuration Files

```
resource "aws_mwaa_environment" "airflow" {

  name                  = var.airflow_service_name
  execution_role_arn    = var.airflow_execution_role
  source_bucket_arn     = "arn:aws:s3:::${var.apps_bucket}"
  dag_s3_path           = "dags/"
  endpoint_management   = "CUSTOMER"

  network_configuration {
    security_group_ids = var.airflow_security_group_ids
    subnet_ids         = var.airflow_subnet_ids
  }
}
```

### Steps to Reproduce

add attribute **endpoint_management**   = **"CUSTOMER"** in aws_mwaa_environment resource

### Important Factoids

MWAA required VPC endpoints needs to be managed by CUSTOMER. Selecting attribute endpoint_management equal to CUSTOMER, mwaa enter in PENDING state during creation, that is not supported.

### Would you like to implement a fix?

No

**Comment by preetsindhal** [comment]
Hi @maxmuffin 
did you use any workaround for now? Something like making terraform wait for a specific timeout after it initiated the mwaa creation?

Thanks

**Comment by maxmuffin** [comment]
Hi @preetsindhal ,
for now the resource has been created manually from the AWS console and imported directly into the terraform state to manage at least the update and delete operations.

**Comment by maxmuffin** [comment]
Hi @preetsindhal ,
for now the resource has been created manually from the AWS console and imported directly into the terraform state to manage at least the update and delete operations.

**Comment by maxmuffin** [comment]
Solved in [https://github.com/hashicorp/terraform-provider-aws/issues/39388](#399388) and released in provider version **5.68.0**
